files/class-vip-filesystem-stream-wrapper.php

```diff
@@ -543,7 +543,7 @@ public function rename( $path_from, $path_to ) {
 			}
 
 			// Convert to actual file to upload to new path
-			$file     = $this->string_to_resource( $result, 'r' );
+			$file     = $this->string_to_resource( file_get_contents( $result ), 'r' );
```

Comment:
`Api_Client:: get_file` returns a `tempnam` path, so we need to `fopen` it:

```suggestion
			$file     = fopen( $result, 'r' );
```

Regression from #1397